docker-compose.yml

```diff
@@ -179,10 +177,7 @@ services:
             - public-api:/app/api/public
             - public-browser:/app/browser/public
         ports:
-            - ${PUBLIC_PORT_BROWSER}:80
-            - ${PUBLIC_PORT_PATTERN_LIBRARY}:8082
-            - ${PUBLIC_PORT_DUMMY_API}:8083
-            - ${PUBLIC_PORT_JATS_INGESTER}:8085
+            - ${PUBLIC_PORT_HTTP}:80
```

Comment:
This means I can't access this in my browser, right? I've found that to be useful...

Reply:
Experimenting with some network magic to provide hostnames locally.

Reply:
Added a DNS server that overrides the host's resolution, will have to test on Macs to see if it works at all.

Reply:
Quick testing reveals this doesn't work on Mac, at least out of the box.

Reply:
https://github.com/mageddo/dns-proxy-server/issues/44